.github/workflows/main.yaml

```diff
@@ -129,9 +129,19 @@ jobs:
       - name: Build Wasmer binary
         run: |
           make build-wasmer
-          # TODO: build wapm
-          # make build-wapm
+      - name: Build Wapm binary
+        run: |
+          make build-wapm
```

Comment:
Note: build-wapm is currently empty I think. Might be worth to add a comment for now

Reply:
It seems to be building wapm in my tests on my fork